src/cli.rs

```diff
@@ -118,6 +118,33 @@ pub struct WatchOptions {
     pub once: bool,
 }
 
+/// Options for the `daemon` subcommand
+#[derive(StructOpt, Debug)]
+pub struct DaemonOptions {
+    #[structopt(
+        long = "extra-nix-options",
+        parse(try_from_str = "serde_json::from_str")
+    )]
+    /// JSON value of nix config options to add.
+    /// Only a subset is supported:
+    /// {
+    ///   "builders": <optional list of string>,
+    ///   "substituters": <optional list of string>
+    /// }
+    pub extra_nix_options: Option<NixOptions>,
+}
+
+/// The nix options we can parse as json string
+#[derive(Deserialize, Debug)]
+// ATTN: If you modify this,
```

Comment:
"ATTN"?

Reply:
“attention”

Similar to TODO and XXX, but if you want to get a developer to hopefully read the comment before modifying the code :)